projects/ngx-charts-on-fhir/src/lib/data-layer-toolbar/data-layer-toolbar/data-layer-toolbar.component.css

```diff
@@ -6,3 +6,8 @@
   width: 48px;
   padding-top: 16px;
 }
+
+#badge-movement ::ng-deep .mat-mdc-tooltip-trigger {
+  padding: 12px 15px 12px 4px;
+}
```

Comment:
I don't think this is the best solution. The buttons are still bigger than the toolbar so the ripple animation overlaps the edge of the toolbar and is not centered on the icon. Can't we just increase the toolbar width so we don't need to modify the material component internals?